app/assets/javascripts/components/common/AssignmentLinks/AssignmentLinks.jsx

```diff
@@ -28,16 +28,14 @@ const AssignmentLinks = ({ assignment, courseType, user }) => {
     actions.push(
       <BibliographyLink key={`bibliography-${id}`} assignment={assignment} />
     );
-  }
-
-  actions.push(
-    <SandboxLink key={`sandbox-${id}`} assignment={assignment} />
-  );
-
-  if (role === REVIEWING_ROLE) {
     actions.push(
-      <PeerReviewLink key={`review-${id}`} assignment={assignment} user={user} />
+      <SandboxLink key={`sandbox-${id}`} assignment={assignment} />
```

Comment:
`.push()` can take multiple arguments, so you could combine this call with the one above!

Reply:
Okay, but how to do feel about `unshift`? After talking with Ryan, I had to make some changes; we always want the sandbox link, but not always the peer review link.

Reply:
Sure thing, that makes sense.